src/datasets/utils/resources/tasks.json

```diff
@@ -1,54 +1,115 @@
 {
-    "conditional-text-generation": {
-        "description": "data-to-text and text transduction tasks such as translation or summarization",
-        "options": [
-            "machine-translation",
-            "sentence-splitting-fusion",
-            "summarization",
-            "table-to-text",
-            "text-simplification",
-            "explanation-generation",
-            "other-structured-to-text",
-            "other"
+    "audio-classification": {
+        "type": "audio",
+        "subtasks": [
+            "keyword-spotting",
+            "speaker-identification",
+            "audio-intent-classification",
+            "audio-emotion-recognition",
+            "audio-language-identification"
         ]
     },
-    "question-answering": {
-        "description": "question answering tasks",
-        "options": [
-            "open-domain-qa",
-            "closed-domain-qa",
+    "audio-to-audio": {
+        "type": "audio"
+    },
+    "automatic-speech-recognition": {
+        "type": "multimodal"
+    },
+    "conversational": {
+        "type": "text",
+        "subtasks": [
+            "dialogue-generation"
+        ]
+    },
+    "feature-extraction": {
+        "type": "multimodal"
+    },
+    "fill-mask": {
+        "type": "text",
+        "subtasks": [
+            "slot-filling",
+            "masked-language-modeling"
+        ]
+    },
+    "image-classification": {
+        "type": "image",
+        "subtasks": [
+            "multi-label-image-classification",
+            "multi-class-image-classification"
+        ]
+    },
+    "image-segmentation": {
+        "type": "image",
+        "subtasks": [
+            "instance-segmentation",
+            "semantic-segmentation",
+            "panoptic-segmentation"
+        ]
+    },
+    "image-to-text": {
+        "type": "multimodal",
+        "subtasks": [
+            "image-captioning"
+        ]
+    },
+    "multiple-choice": {
+        "type": "text",
+        "subtasks": [
             "multiple-choice-qa",
+            "multiple-choice-coreference-resolution"
+        ]
+    },
+    "object-detection": {
+        "type": "image",
+        "subtasks": [
+            "face-detection",
+            "vehicle-detection"
+        ]
+    },
+    "question-answering": {
+        "type": "text",
+        "aliases": [
+            "extractive-question-answering"
+        ],
+        "subtasks": [
             "extractive-qa",
-            "abstractive-qa",
-            "other"
+            "open-domain-qa",
+            "closed-domain-qa"
         ]
     },
-    "sequence-modeling": {
-        "description": "such as language modeling or dialogue",
-        "options": [
-            "dialogue-modeling",
-            "language-modeling",
-            "other-multi-turn",
-            "slot-filling",
-            "other"
+    "sentence-similarity": {
+        "type": "text"
+    },
+    "tabular-classification": {
+        "type": "text",
```

Comment:
I don't think tabular would be text modality no?

Reply:
Yea I'll update it accordingly to https://github.com/huggingface/hub-docs/pull/95 once it's merged